crates/sui-package-resolver/src/lib.rs

```diff
@@ -197,6 +228,9 @@ pub enum OpenSignatureBody {
 struct ResolutionContext<'l> {
     /// Definitions (field information) for structs referred to by types added to this context.
     structs: BTreeMap<DatatypeKey, StructDef>,
+    /// Definitions (variant and field information) for enums referred to by types added to this
```

Comment:
`ResolutionContext` thinks its special and it doesn't get newlines, but it's wrong.